api/tests/routes/backoffice_v3/accounts_test.py

```diff
@@ -495,17 +499,26 @@ def test_fraud_check_link(self, authenticated_client):
             in new_dms_card
         )
 
-    def test_get_public_account_history_date(self, authenticated_client):
+    def test_get_public_account_history(self, legit_user, authenticated_client):
         # given
-        user = users_factories.BeneficiaryGrant18Factory()
+        # More than 30 days ago to have deterministic order because "Import ubble" is generated randomly between
+        # -30 and -1d in BeneficiaryImportStatusFactory
```

Comment:
-30 and -1 days ?

Reply:
Oui, dans `BeneficiaryImportStatusFactory`:
`date = factory.Faker("date_time_between", start_date="-30d", end_date="-1d")`